lib/ap/plan9/malloc.c

```diff
@@ -74,7 +74,7 @@ malloc(size_t size)
 	if(pow < CUTOFF) {
 		n = (CUTOFF-pow)+2;
 		bp = sbrk(size*n);
-		if((intptr_t)bp == -1)
+		if((long)bp == -1)
```

Comment:
Because intptr_t is long long (int64_t). And this seems to be written with 32 integer types.